stream_alert_cli/lookup_tables/handler.py

```diff
@@ -64,10 +183,27 @@ def _setup_describe_tables_subparser(cls, subparsers):
             )
         )
 
+    # pylint: disable=protected-access
     @classmethod
-    def _setup_get_subparser(cls, subparsers):
+    def handler(cls, options, config):
+        print('==== LookupTables; Describe Tables ====\n')
+
+        lookup_tables = LookupTables.get_instance(config=config)
+
+        print('{} Tables:\n'.format(len(lookup_tables._tables)))
+        for _, table in lookup_tables._tables.items():
```

Comment:
suggestion: implement a `__str__` or `__repr__` function on the lookup tables class(es) that can just handle this for you. that way it could also be "neatly" printed elsewhere

Reply:
I'll add a FIXME and cut a new branch for this improvement. I've been thinking of how to refactor the command classes into something that nests more easily